src/cmd/root.go

```diff
@@ -26,20 +26,40 @@ import (
 var (
 	// Default global config for the packager
 	pkgConfig = types.PackagerConfig{}
+	// LogLevelCLI holds the log level as input from a command
+	LogLevelCLI string
+	// SkipLogFile is a flag to skip logging to a file
+	SkipLogFile bool
+	// NoColor is a flag to disable colors in output
+	NoColor bool
 )
 
 var rootCmd = &cobra.Command{
 	Use: "zarf COMMAND",
-	PersistentPreRun: func(cmd *cobra.Command, _ []string) {
+	PersistentPreRunE: func(cmd *cobra.Command, _ []string) error {
 		// Skip for vendor-only commands
 		if common.CheckVendorOnlyFromPath(cmd) {
-			return
+			return nil
 		}
-		// Don't log the help command
+
+		skipLogFile := SkipLogFile
+
+		// Dont write tool commands to file.
+		comps := strings.Split(cmd.CommandPath(), " ")
+		if len(comps) > 1 && comps[1] == "tools" {
```

Comment:
We should also check if the command is version, since we deleted the check PersistentPreRun from there

Reply:
I can add it but in reality it has no effect at all. Just setting it to false is not enough because use of global variables.